# se dara un primer vistaso al comportamiento de las varibales

In [1]:
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
import seaborn as sns

# datos

In [4]:
dfp.columns

Index(['tipo', 'url', 'titulo', 'precio', 'area', 'habitaciones', 'banos',
       'garajes', 'descripcion', 'balcon', 'trans. publico cercano', 'estrato',
       'antiguedad', 'cocina integral', 'bano auxiliar', 'zona de lavanderia',
       'zonas verdes', 'supermercados / c.comerciales', 'instalacion de gas',
       'parqueadero visitantes', 'colegios / universidades',
       'parques cercanos', 'zona residencial', 'closet', 'nombre municipio',
       'codigo departamento', 'codigo municipio', 'nombre departamento',
       'tipo: municipio / isla / area no municipalizada'],
      dtype='object')

In [6]:
dfp.shape

(3480, 29)

## explicacion base de datos

In [10]:
dfp['tipo'].unique()

array([0, 1], dtype=int64)

el dataframe consta de 3480 filas y 29 columnas, las cuales son
* tipo: indica si la vivineda es usada o nueva, donde usada es igual a 0 y nueva igual a 0. es una variable categorica


In [ ]:
# !git clone git@github.com:juanguzman2/Precio-de-viviendas-en-colmbia.git

In [9]:
dfp=pd.read_csv(r'C:\Users\Juan Guzman\pviviendas\Data\dfclean')
dfp.head()

tipo                                                url  \
0     0  https://www.fincaraiz.com.co/casa-campestre-en...   
1     0  https://www.fincaraiz.com.co/casa-campestre-en...   
2     1  https://www.fincaraiz.com.co/vivari-de-san-nic...   
3     1  https://www.fincaraiz.com.co/vivari-de-san-nic...   
4     1  https://www.fincaraiz.com.co/altos-de-santana/...   

                                              titulo      precio     area  \
0  Casa Campestre en Venta Rionegro Vereda Guayabito  1380000000  32000.0   
1  Casa Campestre en Venta Rionegro Vereda Guayabito  1380000000  32000.0   
2       VIVARI DE SAN NICOLAS San Nicolás - Rionegro   496122600  91972.0   
3       VIVARI DE SAN NICOLAS San Nicolás - Rionegro   496122600  91972.0   
4     ALTOS DE SANTANA Oriente Antioqueño - Rionegro   360002000  73372.0   

   habitaciones  banos  garajes  \
0           5.0    6.0        2   
1           5.0    6.0        2   
2           3.0    3.0        1   
3           3.0    3.0        1   
4           3.0    3.0        0   

                                         descripcion  balcon  ...  \
0  Propiedad moderna con muy buenos acabados. Her...     1.0  ...   
1  Propiedad moderna con muy buenos acabados. Her...     1.0  ...   
2  6234794 Vivari San Nicolás, es un conjunto res...     1.0  ...   
3  6234794 Vivari San Nicolás, es un conjunto res...     1.0  ...   
4  6351568 Proyecto de apartamentos de 2 y 3 habi...     0.0  ...   

   parqueadero visitantes  colegios / universidades  parques cercanos  \
0                     0.0                       0.0               0.0   
1                     0.0                       0.0               0.0   
2                     0.0                       0.0               0.0   
3                     0.0                       0.0               0.0   
4                     0.0                       1.0               0.0   

   zona residencial  closet  nombre municipio  codigo departamento  \
0               0.0     0.0          RIONEGRO                    5   
1               0.0     0.0          RIONEGRO                   68   
2               0.0     0.0          RIONEGRO                    5   
3               0.0     0.0          RIONEGRO                   68   
4               0.0     0.0          RIONEGRO                    5   

   codigo municipio  nombre departamento  \
0            5615.0            ANTIOQUIA   
1           68615.0            SANTANDER   
2            5615.0            ANTIOQUIA   
3           68615.0            SANTANDER   
4            5615.0            ANTIOQUIA   

   tipo: municipio / isla / area no municipalizada  
0                                        Municipio  
1                                        Municipio  
2                                        Municipio  
3                                        Municipio  
4                                        Municipio  

[5 rows x 29 columns]

In [3]:
dfp.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3480 entries, 0 to 3479
Data columns (total 29 columns):
 #   Column                                           Non-Null Count  Dtype  
---  ------                                           --------------  -----  
 0   tipo                                             3480 non-null   object 
 1   url                                              3480 non-null   object 
 2   titulo                                           3480 non-null   object 
 3   precio                                           3480 non-null   int64  
 4   area                                             3480 non-null   float64
 5   habitaciones                                     3480 non-null   float64
 6   banos                                            3480 non-null   float64
 7   garajes                                          3480 non-null   int64  
 8   descripcion                                      3480 non-null   object 
 9   balcon                        